cmd/satellite/main.go

```diff
@@ -9,11 +9,9 @@ import (
 	"path/filepath"
 
 	"github.com/spf13/cobra"
-	"storj.io/storj/pkg/cfgstruct"
 
-	// "storj.io/storj/pkg/datarepair/repairer"
-	// "storj.io/storj/pkg/datarepair/checker"
```

Comment:
Did you remove these commented imports intentionally?

Reply:
Yes they mess up goimports.